tuned/plugins/plugin_video.py

```diff
@@ -80,6 +80,8 @@ def _set_radeon_powersave(self, value, device, sim):
 	@command_get("radeon_powersave")
 	def _get_radeon_powersave(self, device, ignore_missing = False):
 		sys_files = self._radeon_powersave_files(device)
+		if not os.path.exists(sys_files["method"]):
+			return None
```

Comment:
The message regarding the missing `power_method` file could be made into a warning instead of an error, however I don't think it makes sense to drop it altogether. The user should be able to learn that Tuned failed to apply the profile, whatever the reason for that is.

Note that the real solution to make the error message go away is to override the Tuned profile and drop the `radeon_powersave` tuning in the overriden profile.